spec/swagger_helper.rb

```diff
@@ -56,26 +58,18 @@
           Application: APPLICATION[version],
           ApplicationAcceptRequest: APPLICATION_ACCEPT_REQUEST,
           ApplicationChangeFundedPlaceRequest: APPLICATION_CHANGE_FUNDED_PLACE_REQUEST,
-        },
+          ApplicationsCsvResponse: APPLICATIONS_CSV_RESPONSE[version],
```

Comment:
just a note that this has been done here https://github.com/DFE-Digital/npq-registration/pull/1400, could we make sure those align as that will probably be merged before?

Reply:
Hi @cwrw Yeah I copied the same changes from there.. 👍